### ISSUE TITLE
Any interest in a watch option?

### ISSUE THREAD

**Issue by mmase** [sasstools/sass-lint#912]

**Comment by DanPurdy** [comment]
No, I don't think so. That should be the job of task runners etc. Sass-lint should be providing only minimal API surface for other tools and task runners to hook into.

**Comment by mmase** [comment]
Alright, that makes sense. I've been seeing a lot of people trying to do away with job runners altogether, us included. But no reason you couldn't still make a standalone script to handle a watch task. Thanks though!

**Comment by DanPurdy** [comment]
Fair enough. I don't think adding a watch task here accomplishes that, it would just add another level of complexity and specificity to how Sass-lint operates which to the majority of our users is probably unwanted and unnecessary. 

I would say what happened with eslint watch here is probably the preferred method. If you want to build something similar we'd be happy to link to it in the Readme https://www.npmjs.com/package/eslint-watch

**Comment by Ricki-BumbleDev** [comment]
I know the issue is very old and you are supposed to use `stylelint` now, but just in case somebody stumbles upon this again like myself: 

`eslint-watch` uses a package called `chokidar` to listen for file changes. There is also a package called `chokidar-cli` which will allow you to write your own `sasslint-watch` in just one line as an npm script:
```sh
chokidar "./src/**/*.s{a,c}ss" -c "sass-lint --no-exit --verbose ./src/**/*.s{a,c}ss" --initial --silent
```

And if you also use ESLint and now use `chokidar-cli` for SASS Linting you might as well also use it for ESLint and drop `eslint-watch`. This is my complete dev setup in the package.json `scripts` section:
```json
    "dev": "npm-run-all --parallel dev:*",
    "dev:eslint": "chokidar \"./src/**/*.{j,t}s{,x}\" -c \"eslint ./src/**/*.{j,t}s{,x}\" --initial --silent",
    "dev:sasslint": "chokidar \"./src/**/*.s{a,c}ss\" -c \"sass-lint --no-exit --verbose ./src/**/*.s{a,c}ss\" --initial --silent",
    "dev:webpack": "webpack-dev-server",
```
